code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -144,6 +144,9 @@ proc/issyndicate(mob/living/M as mob)
 	var/mob/living/carbon/human/M = synd_mind.current
 	var/obj/item/organ/external/head/head_organ = M.get_organ("head")
 
+	M.mutations = list()
+	M.disabilities = 0
+	M.overeatduration = 0

```

Comment:
The `disabilities` bitfield is in general really bad - you might want to use the `CureATTRIBUTE` functions instead to ensure things like blindness and similar instantly update, and I think there's also a `becomeSlim` proc that will make someone normal, and also make their torso organ normal as well. (though I think `becomeSlim` gives a message, too, so perhaps a silence parameter would be wise)